content/docs/latest/agentgateway/llm/providers/bedrock.md

```diff
@@ -25,10 +25,10 @@ Configure [Amazon Bedrock](https://aws.amazon.com/bedrock/) as an LLM provider i
 
    ```yaml
    kubectl create secret generic bedrock-secret \
-     -n {{< reuse "docs/snippets/namespace.md" >}} \
-     --from-literal=aws_access_key_id="$AWS_ACCESS_KEY_ID" \
-     --from-literal=aws_secret_access_key="$AWS_SECRET_ACCESS_KEY" \
-     --from-literal=aws_session_token="$AWS_SESSION_TOKEN" \
+     -n kgateway-system \
```

Comment:
Would you switch all the kgateway-system namespaces back to the namespace variable, `{{< reuse "docs/snippets/namespace.md" >}}`?